effekt/shared/src/main/scala/effekt/core/PolymorphismBoxing.scala

```diff
@@ -198,7 +198,7 @@ object PolymorphismBoxing extends Phase[CoreTransformed, CoreTransformed] {
       if (coerce.isIdentity) {
         List(Definition.Let(id, transform(tpe), transform(binding)))
       } else {
-        val orig = TmpValue()
+        val orig = TmpValue("toBeCoerced")
```

Comment:
I think it's also fine to have just something like `coe` there, especially since the result looks like `v_coe_1234` (as opposed to `v_toBeCoerced_1234`, but again, I'm personally fine with both.